YSI_Data/y_foreach/features.md

```diff
@@ -135,7 +135,7 @@ for (new i = -1; (i = FUNC(i)) != -1; )
 
 As a side note, this won't work for regular iterators because `-1` is not a
 valid array index, so can't be used as the start value without a significant
-loss in efficiency.
+loss inefficiency.

```

Comment:
A loss in inefficiency would be a good thing, but we are avoiding a loss of "efficiency", so the original was correct.